src/Lib/Network/SocketBase.cpp

```diff
@@ -103,7 +103,7 @@ SocketBase::create ()
     {
         sockfd = socket(PF_INET, addr.socktype, addr.protocol);
 
-        LOGGER.debug("SocketBase:: Create [%s:%lld] socket", (addr.socktype == SOCK_STREAM)?"tcp":"udp",sockfd);
```

Comment:
I need to test this on windows. I think on win64 sockfd is "long long" or whatever

Reply:
k, I removed that change.